packages/ui-extensions-react/src/surfaces/customer-account/hooks/extension-data.ts

```diff
@@ -1,15 +0,0 @@
-import type {
-  Extension,
-  RenderExtensionTarget,
-} from '@shopify/ui-extensions/customer-account';
-
-import {useApi} from './api';
-
-/**
```

Comment:
why this is removed? did we moved it somewhere else? if checkout added it, we should keep it maybe